ui/raidboss/data/05-shb/dungeon/qitana_ravel.js

```diff
@@ -309,5 +310,37 @@
         'Viper Poison': 'Viperngift',
       },
     },
+    {
+      'locale': 'ko',
+      'replaceSync': {
+        'Batsquatch': '배츠콰치',
+        'Eros': '에로스',
+        'Lozatl': '로차틀',
+        '00:0839:The Divine Threshold will be sealed off': '초 후에 신에게 가는 문이 봉쇄됩니다',
```

Comment:
My only other comment here is that it'd be better not to have the `00:0839` as part of the replacement.  For both Korean and German, it'd be better to just have this style: `'The Divine Threshold will be sealed off': '초 후에 신에게 가는 문이 봉쇄됩니다',` so that the 00:0839 (which is part of the line) is still there.

Reply:
ex) Original EN sync:
`00:0839:The Divine Threshold will be sealed off **in 15 seconds**`

but in KO, DE
KO - 00:0839:**15**초 후에 신에게 가는 문이 봉쇄됩니다
DE - 00:0839:**Noch 15** Sekunden bis sich der Zugang zu Götterpforte schließt

like this, the seconds comes in front of the message.

I think if only back part after '00:0839:' is replaced, the changed message will be
`00:0839:초 후에 신에게 가는 문이 봉쇄됩니다`
and it is not synced with 00:0839:**15**초 후에 신에게 가는 문이 봉쇄됩니다

Writing this comment, I realize I was stupid and there was another good way...
I'll replace these sync texts to include second texts.